visidata/features/fill.py

```diff
@@ -17,7 +17,7 @@ def fillNullValues(vd, col, rows):
             val = e
 
         if isNull(val):
-            if lastval and (id(r) in rowsToFill):
+            if lastval is not None and (id(r) in rowsToFill):
```

Comment:
Good catch, @midichef!  But let's use `if not isNull(lastval)`, so that the user can define their own null via `options.null_value`.

Reply:
Oh, so that is what `isNullFunc` is for! Thanks.